mw-webapp/src/logic/wayPage/WayPage.tsx

```diff
@@ -59,6 +61,17 @@ export const WayPage = (props: WayPageProps) => {
             onChangeFinish={(text) => changeWayName(way, text)}
             isEditable={true}
           />
+          <EditableTextarea
+            text={way.goal.description}
+            onChangeFinish={(description) => {
+              const goalDTO = {
+                ...way.goal,
```

Comment:
We don't use DTO in UI layer. Looks like it's wrong naming in your case (goalDTO => goal). Please check does you understand models right) If you  have questions call me in Telegram

Reply:
Done.